src/routes/cfp/+page.svelte

```diff
@@ -168,6 +182,9 @@
     }
   }
 
+  const validateEmail = (mail: string) =>
+    /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(mail)
```

Comment:
I recommend avoiding regex to validate e-mail. This one, for instance, doesn't support TLD with more than three characters, [and it's valid](https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains).
Also, it's possible to have email with accents. Gmail allowed it in the past and there are many users still using a gmail address with accents.

Just to avoid these edge cases, please, pick a validation library with good support for email.